homeassistant/components/updater/__init__.py

```diff
@@ -95,15 +96,24 @@ def _load_uuid(hass, filename=UPDATER_UUID_FILE):
             newest = hass.components.hassio.get_homeassistant_version()
 
         # Validate version
+        update_available = False
         if StrictVersion(newest) > StrictVersion(current_version):
             _LOGGER.info("The latest available version is %s", newest)
-            hass.states.async_set(
-                ENTITY_ID, newest, {ATTR_FRIENDLY_NAME: 'Update Available',
-                                    ATTR_RELEASE_NOTES: releasenotes}
-            )
+            update_available = True
         elif StrictVersion(newest) == StrictVersion(current_version):
             _LOGGER.info(
                 "You are on the latest version (%s) of Home Assistant", newest)
+        elif StrictVersion(newest) < StrictVersion(current_version):
+            _LOGGER.warning(
```

Comment:
This should not be a warning.

Reply:
I made this a warning because everyone should know (and notice) that the information about the latest version (https://updater.home-assistant.io/) may be outdated and the updater component works well. Also, this is a state that should not normally occur.

But if you don't want this to be a warning, I can change it to Info.

```suggestion
            _LOGGER.info(
```

Reply:
If you don't agree with me, please click on 'Commit suggestion'. I am also happy if this message is an info.

Reply:
@balloob Sorry for bumping up that PR again. I guess you're very busy at the moment. But I would love to see these changes in the next versions of Home Assistant.  If there is anything I can do to speed up this review, please tell me. Thank you

Reply:
Log it debug. We should fix the updater and not put workarounds in for things that we control ourselves.

Reply:
Done